packages/zowe-explorer-api/src/extend/MainframeInteraction.ts

```diff
@@ -119,13 +119,23 @@ export namespace MainframeInteraction {
          * @param {string} ussFilePath
          * @param {zosfiles.IDownloadOptions} options
          */
-        getContents(ussFilePath: string, options: zosfiles.IDownloadOptions): Promise<zosfiles.IZosFilesResponse>;
+        getContents(ussFilePath: string, options: zosfiles.IDownloadSingleOptions): Promise<zosfiles.IZosFilesResponse>;
 
         /**
          * Uploads a given buffer as the contents of a file on USS.
          */
         uploadFromBuffer(buffer: Buffer, filePath: string, options?: zosfiles.IUploadOptions): Promise<zosfiles.IZosFilesResponse>;
 
+        /**
+         * Uploads a given buffer as the contents of a file on USS.
+         */
+        uploadFromBuffer(buffer: Buffer, filePath: string, options?: zosfiles.IUploadOptions): Promise<string | zosfiles.IZosFilesResponse>;
```

Comment:
is it necessary to keep both instances?
Or could we just note the possible breaking change of the return type being a `string` too 😋

Reply:
Nope - sorry, this was supposed to be replaced during a merge conflict and I didn't catch it.
Addressed in b5d9d3a1